controller/kubernetes_pod_controller.go

```diff
@@ -128,6 +128,10 @@ func (kc *KubernetesPodController) handleErr(err error, key interface{}) {
 	utilruntime.HandleError(err)
 }
 
+func getLoggerForPod(logger logrus.FieldLogger, pod *v1.Pod) *logrus.Entry {
```

Comment:
nit: I suggest we can refactor this kind of getting logger in a common place, like util.

Reply:
https://github.com/longhorn/longhorn/issues/5102